### PR TITLE
add wordpress webhook tutorial

### PR DESCRIPTION
**Pull request by bcho**

### CODE REVIEW

**Comment by bcho** [comment]
@loddit @yuanbohan cc @zjhmale

File: robots/wordpress.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Bearychat WordPress Webhooks 手动配置指南
```
Comment (yuanbohan) [inline robots/wordpress.md]: BearyChat

**Comment by yuanbohan** [comment]
LGTM

**Comment by loddit** [comment]
LGTM

**Pull request merged**